zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```
@@ -3412,6 +3413,115 @@ paths:
         "500":
           $ref: "#/components/responses/InternalServerError"
 
+  /batch-operations/{batchOperationKey}:
+    get:
+      tags:
+        - Batch Operation
+      operationId: getBatchOperation
+      summary: Get Batch Operation for batch operation key
+      description: Get Batch Operation
+      parameters:
+        - name: batchOperationKey
+          in: path
+          required: true
+          description: |
+            The key of the batch operation
+          schema:
+            type: string
+      responses:
+        "200":
+          description: Batch Operation
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/BatchOperationResponse"
+        "400":
+          description: The provided data is not valid.
+          content:
+            application/problem+json:
+              schema:
+                $ref: "#/components/schemas/ProblemDetail"
+        "404":
+          description: The signal is not found.
+          content:
+            application/problem+json:
+              schema:
+                $ref: "#/components/schemas/ProblemDetail"
+        "500":
+          $ref: "#/components/responses/InternalServerError"
+
+  /batch-operations/{batchOperationKey}/items:
+    get:
+      tags:
+        - Batch Operation Items
+      operationId: getBatchOperationItems
+      summary: Get Items for Batch Operation
```

```suggestion
      summary: Get batch operation items
```

🔧 Casing
🔧 Following the pattern of `process instance variables`, specify the primary item followed by its associated items, rather than the reverse.